disnake/webhook/async_.py

```diff
@@ -960,6 +961,14 @@ def is_authenticated(self) -> bool:
         """
         return self.auth_token is not None
```

Comment:
<sup>Thank you very much github for not allowing me to comment on any line. Actually, no, f*ck you, the feature request for this is dangling ever since 2021 and you did nothing about it</sup>

The `Webhook.is_partial` method just a bit above should be removed.